src/theme/typography.ts

```diff
@@ -0,0 +1,5 @@
+import { TypographyOptions } from "@mui/material/styles/createTypography";
+
+const typography: TypographyOptions = {};
```

Comment:
the same goes here for the typography export

Reply:
resolved